web/src/components/documentations/DocumentationCreateForm.vue

```diff
@@ -27,16 +28,18 @@
         <v-text-field
           v-model="documentation.description"
           label="Description"
+          hide-details
         />
       </v-col>
       <v-col
         cols="12"
         md="6"
       >
-        <documentation-format-select
+        <DocumentationFormatSelect
           v-model="documentation.format"
           label="Format"
           :rules="[required]"
+          hide-details
```

Comment:
If you find you are using this everywhere, it might be worth setting it at the Vuetify plugin level.